.eslintrc

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Note: the latest version of `eslint-config-brightspace` has a Polymer config that extends the eslint and D2L rules, and includes the globals, which would reduce this to:
```
{
  "extends": "brightspace/polymer-config"
}
```
If you decide to use it, be note that you will still need to reference the `html` plugin in `package.json` though.